libraries/stdlib/genosl/lib/vector4_extra_ops.osl

```diff
@@ -3,10 +3,7 @@
 
 vector4 __operator__mul__(matrix m, vector4 v)
 {
-  return vector4(v.x * m[0][0] + v.y * m[0][1] + v.z * m[0][2] + v.w * m[0][3], 
-                 v.x * m[1][0] + v.y * m[1][1] + v.z * m[1][2] + v.w * m[1][3], 
-                 v.x * m[2][0] + v.y * m[2][1] + v.z * m[2][2] + v.w * m[2][3], 
-                 v.x * m[3][0] + v.y * m[3][1] + v.z * m[3][2] + v.w * m[3][3]);
+  return transform(m, v);
```

Comment:
these changes here to the OSL functions ~ they are unrelated to the OCIO integration, right? I think it would make sense to commit them independently so as not to conflate unrelated changes in one commit?

Reply:
This file is introduced by this pull request. Would be nice if it was correct on the first try. Latest changes recognizes the OSL multiplication order as defined in `transform`.